tangelo/toolboxes/molecular_computation/tests/test_rdms.py

```diff
@@ -130,6 +131,22 @@ def test_compute_rdms_from_classical_shadow(self):
         assert_allclose(rdm1ssr, rdm1ss, atol=0.05)
         assert_allclose(rdm2ssr, rdm2ss, atol=0.05)
 
+    def test_pad_rdms_with_frozen_orbitals(self):
+        """Test padding of RDMs with frozen orbitals indices."""
+
+        mol = mol_H2O_sto3g.freeze_mos([0, 3, 4, 5], inplace=False)
+
+        onerdm_to_pad = np.loadtxt(pwd_this_test + "/data/H2O_sto3g_onerdm_frozen0345.data")
+        twordm_to_pad = np.loadtxt(pwd_this_test + "/data/H2O_sto3g_twordm_frozen0345.data")
+
+        test_onerdm, test_twordm = pad_rdms_with_frozen_orbitals(mol, onerdm_to_pad, twordm_to_pad.reshape((3,)*4))
+
+        padded_onerdm = np.loadtxt(pwd_this_test + "/data/H2O_sto3g_padded_onerdm_frozen0345.data")
+        padded_twordm = np.loadtxt(pwd_this_test + "/data/H2O_sto3g_padded_twordm_frozen0345.data")
+
+        np.testing.assert_array_almost_equal(padded_onerdm, test_onerdm)
```

Comment:
This data may have a lot of significant digits. I encourage you to explicitly choose and specify the threshold you use for "almost equal" in L147-148, it is not obvious what it is.